workos/users.py

```diff
@@ -289,3 +289,52 @@ def authenticate_with_password(
         )
 
         return WorkOSAuthenticationResponse.construct_from_response(response).to_dict()
+
+    def authenticate_with_token(
```

Comment:
Let's name this `authenticate_with_code` to match the other SDKs:

```suggestion
    def authenticate_with_code(
```

Reply:
Good catch. The current Node docs got me. 😄